reconciletags/reconciletags_e2e_test.py

```diff
@@ -70,35 +70,42 @@ def setUp(self):
     def tearDown(self):
         subprocess.call(['gcloud', 'container', 'images',
                          'delete', self._REPO + '@sha256:' + self.digest,
-                         '-q'])
-
-    def testTagReconciler(self):
-        # run the reconciler
-        self.r.reconcile_tags(self._TEST_JSON, False)
+                         '-q', '--force-delete-tags'])
 
-        # check list-tags to see if it added the correct tag
+    def checkListTagsOutput(self):
         output = self._ListTags(self._REPO)
         for image in output:
             if image['digest'].split(':')[1] == self.digest:
                 self.assertEquals(len(image['tags']), 2)
                 self.assertEquals(image['tags'][1], 'testing')
                 self.assertEquals(image['tags'][0], self._TAG)
 
-        # run reconciler again and make sure nothing changed
+    def testTagReconciler(self):
```

Comment:
Does this test pass? This would have to be run through bazel I guess, right?

Reply:
yah, it passes through bazel but not otherwsie!